spec/services/reporting/total_user_count_report_spec.rb

```diff
@@ -16,14 +16,14 @@
 
   before { travel_to report_date }
 
-  shared_examples 'a report with that user counted' do
-    let(:expected_count) { 1 }
-    it 'includes that user in the count' do
-      expect(subject.total_user_count_report).to eq expected_report
```

Comment:
This shared example was specific to `#total_user_count_report` but I didn't initially put it there; this change is just fixing that.